STANDARDS.md

```diff
@@ -8,6 +8,7 @@ This document lists down the standards for OpenSearch plugins which can be used
   - [License Headers](#license-headers)
   - [Public Documentation](#public-documentation)
   - [Release Notes](#release-notes)  
+  - [Code Coverage Report](#code-coverage-report)
```

Comment:
super nit: lets put this in an alphabetical order :)

Reply:
Ah, I didn't realize this rule. I will change it.

Reply:
hidden treasures :D

Reply:
Hidden very deep. 🙂